src/usr/local/www/firewall_virtual_ip_edit.php

```diff
@@ -147,7 +147,7 @@ function find_last_used_vhid() {
 
 		if (isset($network_addr) && $_POST['subnet'] == $network_addr) {
 			$input_errors[] = gettext("The network address cannot be used for this VIP");
-		} else if (isset($broadcast_addr) && $_POST['subnet'] == $broadcast_addr) {
+		} else if (is_ipaddrv4($_POST['subnet']) && isset($broadcast_addr) && $_POST['subnet'] == $broadcast_addr) {
```

Comment:
The same result could be reached just removing line 145 and then $broadcast_addr will never be set for IPv6